### ISSUE TITLE
Confessor items are tedious

### ISSUE THREAD

**Issue by BrandenEK** [BrandenEK/BlasII.Randomizer#156]
Especially with being possible so early on, this sucks.  If I can remove the "ShuffleLongQuests" option by making these not "possible but tedious" early on, that would be ideal.

If we can change his main purpose to just giving items, the confessor prie dieu cherub could also be unlocked from the beginning.

Possibilites:
- Make all of them or at some of the items unlocked from the start (Walk in and grab them)
- Add item gates (Instead of cleansing guilt, you need an item or event to do it)

**Comment by BrandenEK** [comment]
## Suggestion:
From a gameplay perspective, you start the game with his quest already finished.  The confessor prie dieu cherub is already unlocked, so you cleanse guilt there.  When you enter his room, it is already charred and he is already "dead", with the 2 flower items on the floor.  He does not give a third item, so the only purpose to going there is to pick up those 2 items at the beginning, and later on to get the item from the Hand quest.

**Comment by ObsessiveBadguy** [comment]
Yeah i think for the advertised QoL and for the sake of a simpler randomizer this is a solid idea, it gets my approval

**Comment by NeonPixels** [comment]
> He does not give a third item, so the only purpose to going there is to pick up those 2 items at the beginning, and later on to get the item from the Hand quest

Ehm, he does give his remembrance after he is taken over by the evil confessor. Maybe it can be added as a shiny on the floor?
